controllers/administration.js

```diff
@@ -748,13 +748,23 @@ const returnAdminPrefix = (roles) => {
 };
 
 // with userId to accountId
-const userIdtoAccountIdUpdate = (service) => function useIdtoAccountId(req, res, next) {
+const userIdToAccountIdUpdate = (service) => async function useIdtoAccountId(req, res, next) {
+	try {
+		await api(req)
+			.patch(`/users/${req.params.id}`, {
+				json: { forcePasswordChange: true },
```

Comment:
Do you write test how the permission is work in server? Only person with permission in this school, superhero and the user him self should set this key.

Reply:
According to our discussion. No need to do that.